frontend/lib/signup.ts

```diff
@@ -0,0 +1,18 @@
+export const signUp = async (
+  username: string,
+  email: string,
+  password: string
+) => {
+  const response = await fetch("http://localhost:3001/users", {
```

Comment:
Maybe, we can open a new issue to abstract out the `localhost:3001` from all API request strings and put it into a global variable called `userServiceURL`. All other API requests can follow this same format too, so that it is easily to keep track of any changes.

Reply:
Added issue to do this https://github.com/CS3219-AY2425S1/cs3219-ay2425s1-project-g50/issues/99